tools/benchmark/benchmark_codec_avif.cc

```diff
@@ -212,14 +224,16 @@ class AvifCodec : public ImageCodec {
           avifEncoderCreate(), &avifEncoderDestroy);
       encoder->codecChoice = encoder_;
       // TODO(sboukortt): configure this separately.
-      encoder->minQuantizer = encoder->maxQuantizer =
-          AVIF_QUANTIZER_BEST_QUALITY +
-          (AVIF_QUANTIZER_WORST_QUALITY - AVIF_QUANTIZER_BEST_QUALITY) *
-              (1 - 0.01 * q_target_);
```

Comment:
That means that `q_target_` is not used anymore, right?

Reply:
Correct.